### DISCUSSION TITLE
How to create a quirk for a device that already reports values in the correct clusters?

### DISCUSSION THREAD

**Discussion by cremor** [zigpy/zha-device-handlers#1430] [Q&A] [answered]
I bought a window covering (blinds) device that doesn't show tilt values in HA. But I can use the Zigbee attribute "current_position_tilt_percentage" (0x0009) in the cluster "WindowCovering" (0x0102 in endpoint 6) to get the tilt value. So the device seems to report it correctly, but ZHA isn't using it.

How would a quirk for that need to look like? I assume overriding the clusters with the exact same ones that are there originally wouldn't change anything?

Maybe this is related to the problem that it looks like the device is identified as a window? At least it's shown with a window icon. If that is the case, how would I tell ZHA that this is a blind - so that it supports tilt?

It's about this device support request: #1397

**Comment by cremor** [comment, marked as answer]
This question was based on a wrong assumption. ZHA simply does not yet support tilt for covers. So it wouldn't help to create a quirk for the device. More information in the device support request #1397

**Comment by Hedda** [comment]
FYI, check out a related zigpy development discussion here -> https://github.com/zigpy/zigpy/discussions/934

Developers there discuss ideas to expose more generic device configuration attributes in an easy way for ZHA UI/GUI.
